rotkehlchen/inquirer.py

```diff
@@ -342,11 +348,13 @@ def inject_evm_managers(evm_managers: Sequence[tuple[ChainID, 'EvmManager']]) ->
         for chain_id, evm_manager in evm_managers:
             instance._evm_managers[chain_id] = evm_manager
 
-    @staticmethod
-    def get_ethereum_manager() -> 'EthereumManager':
-        ethereum_manager = Inquirer()._evm_managers.get(ChainID.ETHEREUM)
-        assert ethereum_manager is not None, 'ethereum manager should have been injected'
-        return ethereum_manager  # type: ignore  # should be ethereum manager
+    def get_evm_manager(self, chain_id: ChainID) -> 'EvmManager':
+        evm_manager = self._evm_managers.get(chain_id)
+        assert evm_manager is not None, f'evm manager for chain id {chain_id} should have been injected'  # noqa: E501
+        return evm_manager
+
+    def get_ethereum_manager(self) -> 'EthereumManager':
+        return self.get_evm_manager(ChainID.ETHEREUM)  # type: ignore  # should be ethereum manager
```

Comment:
why don't you use `get_evm_manager ` directly in the places where this method is required? Is just a one line wrapper that adds overhead